runtimes/peregrine/src/filter.rs

```diff
@@ -0,0 +1,48 @@
+use frame_support::traits::Contains;
+
+use super::Call;
+
+pub struct TransferCalls;
+impl Contains<Call> for TransferCalls {
+	fn contains(t: &Call) -> bool {
+		matches!(
+			t,
+			Call::Balances(..)
+				| Call::Indices(pallet_indices::Call::force_transfer { .. } | pallet_indices::Call::transfer { .. })
```

Comment:
I am confused by the intended scope of the transfer filter. Why are we allowing the indices calls? Because they result in the release of a deposit? If that is the case, why don't we allow also the reclaim deposit functions from the other pallets?

Reply:
Biggest concern is that people submit multiple transactions because they see that they don't go through. "Oh hey my transfer didn't go through, I'll try again!"
When we start building blocks both transfers are valid and will be execute even tho only one was intended.

This filter disables the whole balances pallet, the indices transfers and vested transfers. It should only filter calls where funds get moved from one account to another.